### PR TITLE
TEST-2022: speed up prepare-cache job

### LINKED ISSUE #2022: Speed up prepare-cache job

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

Speed up prepare-cache job by ~2 minutes.

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #2022  
- [ ] tests added and passing

### CODE REVIEW

**Comment by anmyachev** [comment]
part of #2008

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -70,19 +70,11 @@ jobs:
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
-      - uses: goanpeca/setup-miniconda@v1.6.0
+      - uses: actions/setup-python@v2
         with:
           python-version: ${{matrix.python-version}}
-          channel-priority: strict
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
```
Comment (dchigarev) [inline .github/workflows/ci.yml:77]: we don't need that value anymore?

Comment (anmyachev) [reply .github/workflows/ci.yml:77]: This is used further down the code.

Just to cache pip, we don't need to create a base conda environment. It's slower than using only pip.

**Review by dchigarev — APPROVED** [review]
LGTM! thanks @anmyachev

**Pull request merged**